internal/providers/terraform/azure/registry.go

```diff
@@ -158,6 +159,12 @@ var FreeResources []string = []string{
 	// Azure DNS
 	"azurerm_private_dns_zone_virtual_network_link",
 
+	// Azure Event Hub
```

Comment:
Plz add these to the free list as the namespace one is where we're mapping the costs to.
```
    azurerm_eventhub
    azurerm_eventhub_authorization_rule
    azurerm_eventhub_cluster
    azurerm_eventhub_consumer_group
    azurerm_eventhub_namespace_authorization_rule
    azurerm_eventhub_namespace_customer_managed_key
    azurerm_eventhub_namespace_disaster_recovery_config
```

Reply:
added free resources